apis_core/apis_entities/models.py

```diff
@@ -274,6 +274,26 @@ def get_duplicate_url(self):
             kwargs={"entity": entity, "pk": self.id},
         )
 
+    def merge_fields(self, other):
+        """
+        This method is intended to be extended by subclasses.
+        It is called by the `merge_with` method and should transfer data from `other` to `self`
+        """
+        # Inherited from RootObject
+        if self.name != other.name:
+            self.name = ", ".join([self.name, other.name])
```

Comment:
I think it should be rather
```python
self.name = f"{self.name} ({other.name})"
```
for the dates `self` is overruling `other` when both objects have dates, that should be reflected in the string based fields as well